andela/mont-blanc-cfh
# #151991936 Users should be able to create/start a new game

#### What does this PR do?
+ It implements a feature that prevents the game from starting automatically when the maximum number of required players is reached.
#### Description of Task to be completed?
+ When players get invited to the game, a `start game` button appears when the minimum or the maximum number of required players is reached, but the game only starts when the button is clicked.
#### How should this be manually tested?
+ Observe the app game screen to verify that the game only starts when the `start game` button gets clicked when the minimum or the maximum number of required players is reached.
#### Any background context you want to provide?
+ None.
#### What are the relevant pivotal tracker stories?
`#151991929`
#### Questions:
+ None
